skimage/data/__init__.py

```diff
@@ -7,15 +7,20 @@
 """
 
 import os as _os
+import numpy as np
 
 import numpy as _np
+from distutils.version import LooseVersion as Version
+import pooch
 
 from ..io import imread, use_plugin
 from .._shared._warnings import expected_warnings, warn
 from ..util.dtype import img_as_bool
 from ._binary_blobs import binary_blobs
 from ._detect import lbp_frontal_face_cascade_filename
 
+from .. import __version__
+
 import os.path as osp
 data_dir = osp.abspath(osp.dirname(__file__))
```

Comment:
data_dir is  basically useless now.

The only migration path I can see, is to create some really hacky class, that implements the 

```
class _data_dir:
    def __init__(self, image_fetcher):
         self.image_fetcher = image_fetcher
    def __str__(self):
          warn(....)
          prefetch()
          return str(self.image_fetcher.path)
    def __add__(self, filename):
         if filename in self.image_fetcher.registry:
                warn(.....)
                return self.image_fetcher.fetch(filename)
         else:  # Globbing???
                prefetch()
                return str(self.image_fetcher.path) + filename

data_dir = _data_dir(image_fetcher)
```

Reply:
the usefulness of datadir has been restored